### DISCUSSION TITLE
新人入站，发图。

### DISCUSSION THREAD

**Discussion by jidianzhuxing** [share121/inter-knot#189] [其他]
如题，来点图，既是绳匠也是步枪兵
![脉冲刀4](https://github.com/user-attachments/assets/361b68e6-a803-4467-83e0-a3e79ac2b12e)

**Comment by NOVAQUQ** [comment]
![Screenshot_2024-03-22-02-37-53-94_149003a2d400f6adb210d7e357a3a646](https://github.com/user-attachments/assets/95c17cf6-0f58-45da-b134-abfcca2d815a)
